### PR TITLE
[Enhancement] upgrade isort in pre-commit config

### PR DESCRIPTION
**Pull request by lvhan028**
## Motivation
https://github.com/open-mmlab/mmflow/pull/87

## Modification

1. Remove seed-isort-config in pre-commit hook
2. Upgrade isort to v5.10.1 (latest version)
3. Replace known_standard_library with extra_standard_library

### CODE REVIEW

File: mmdeploy/backend/ncnn/wrapper.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 from mmdeploy.utils import Backend, get_root_logger
 from mmdeploy.utils.timer import TimeCounter
+
```
Comment (AllentDan) [inline mmdeploy/backend/ncnn/wrapper.py:11]: would this change make mmdeploy a third extra library?

**Review by RunningLeon — APPROVED** [review]
LGTM

File: mmdeploy/backend/ncnn/wrapper.py (hunk 1 above)
Comment (lvhan028) [reply mmdeploy/backend/ncnn/wrapper.py]: You mean `extra_standard_library` making mmdeploy a third extra library?

**Review by MeowZheng — APPROVED** [review]

File: mmdeploy/backend/ncnn/wrapper.py (hunk 1 above)
Comment (AllentDan) [reply mmdeploy/backend/ncnn/wrapper.py]: I was meaning that the previous codes made `mmdeploy` a third-party library. Now it gets fixed then

**Review by AllentDan — APPROVED** [review]
LGTM

**Pull request merged**